Easydict/Feature/ViewController/View/ResultView/EZResultView.m

```diff
@@ -361,6 +396,21 @@ - (void)updateArrowButton {
     }];
 }
 
+- (BOOL)isOpenAIService:(EZQueryService *)service {
+    EZServiceType type = service.serviceType;
+    return type == EZServiceTypeCustomOpenAI || type == EZServiceTypeOpenAI;
```

Comment:
Would it be better to judge whether it is an OpenAI type of service? Later we need to support custom OpenAI services https://github.com/tisfeng/Easydict/issues/439#issuecomment-1977974655 .

```objc
[service isKindOfClass:EZOpenAILikeService.class]
```